lib/mirage.ml

```diff
@@ -1517,12 +1588,17 @@ let add_to_opam_packages l =
 
 (** {Custom registration} *)
 
-let register ?tracing ?keys ?(libraries=[]) ?(packages=[]) name jobs =
+let register
+    ?tracing ?(reporter=Some (default_reporter ()))
+    ?keys ?(libraries=[]) ?(packages=[])
+    name jobs =
   let libraries = !libraries_ref @ libraries in
   let packages = !packages_ref @ packages in
-  let jobs = match tracing with
-    | None -> jobs
-    | Some tracing ->
-      tracing :: jobs
+  let jobs = in_parallel jobs in
```

Comment:
Then the current implementation has an issue then. I'll fix https://github.com/mirage/functoria/pull/58#discussion_r61314212